src/registry/billingsdk/usage-forecast.tsx

```
@@ -0,0 +1,405 @@
+                        </CardDescription>
+                    )}
+                </CardHeader>
+            )}
+            
+            <CardContent className={cn("grid grid-cols-1 gap-6", config.spacing)}>
+                {metrics.map((metric, i) => {
+                    const forecast = forecastMetric(metric)
+                    const projectedPercentage = (forecast.projected / metric.limit) * 100
+                    const chartData = generateChartData(metric, forecast)
+
+                    const motionValue = useMotionValue(0)
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)